src/app/_components/GovernanceCalendarCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { Clock, CalendarBlank } from '@phosphor-icons/react/dist/ssr'
+
+import { Heading } from '@/components/Heading'
+
+import { addMinutesToISO, extractTimeFromISO } from '@/utils/convertISOTime'
+
+import { TextLink } from './TextLink'
+
+export type GovernanceCalendarCardProps = {
+  title: string
+  startDate: string
+  cta: {
+    url: string
+    text: string
+  }
+  durationInMinutes: number
+  as?: React.ElementType
+}
+
+type CalendarProps = {
+  startDate: string
+}
+
+function Calendar({ startDate }: CalendarProps) {
+  const { day, month } = extractTimeFromISO(startDate)
+  return (
+    <div className="min-w-xs grid grid-rows-[_40px,_96px] rounded-md border border-blue-400 bg-blue-500 sm:min-w-32">
+      <span className="flex items-center justify-center rounded-tl-[5px] rounded-tr-[5px] text-base font-normal uppercase">
+        {month}
+      </span>
+      <span className="flex items-center justify-center rounded-bl-[5px]  rounded-br-[5px] bg-blue-100 text-5xl font-bold text-blue-400">
+        {day}
+      </span>
+    </div>
+  )
+}
+
+export function GovernanceCalendarCard({
+  title,
+  startDate,
+  durationInMinutes,
+  cta,
+  as: Tag = 'li',
```

Comment:
Just curious, can `as` be anything else than `li`?

Reply:
Good point. Since CardLayout is a `ul`, I can just set the component to be `li,` guess there's no cases where' we'd need to modify it.